File: TEO/Mav.UI/Mav.Ui/src/app/models/exam-question.view.model.ts
import { ExamAnswerViewModel } from './exam-answer.view.model';

export class ExamQuestionViewModel {

    public id: number;

    public groupName: string;

    public number: number;

    public typeID: number;

    public selections: number;

    public title: string;

    public answers: ExamAnswerViewModel[];

}

File: TEO/Mav.Models/ViewModels/ExamQuestionViewModel.cs
using System.Collections.Generic;

namespace Mav.Models.ViewModels
{
    public class ExamQuestionViewModel
    {
        public int ID { get; set; }

        public string GroupName { get; set; }

        public int Number { get; set; }

        public int TypeID { get; set; }

        public int? Selections { get; set; }

        public string Title { get; set; }

        public List<ExamAnswerViewModel> Answers { get; set; }
    }

    public class ExamAnswerViewModel
    {
        public int ID { get; set; }

        public string Title { get; set; }

        public int OrderID { get; set; }
    }
}
